### ISSUE TITLE
Why do we exclude "resolved" but not uplifted bugs in regression counts?

### ISSUE THREAD

**Issue by mnoorenberghe** [mozilla/releasehealth#14]
The two regression bug queries include `&resolution=---` but the status flags are the source of truth about the fix on the specified version and are already part of the query. This means that a regression bug affecting all branches will disappear from the count when it's fixed on Nightly-only. IMO it should only be removed when it's uplifted as uplifts aren't necessarily immediate (e.g. the author may be asked to rebase).

In summary, I think `&resolution=---` should be removed from the two queries to be more accurate about the state of the release (which is what the dashboard is all about IIUC).

**Comment by lmandel** [comment]
You're right. I really thought I had removed resolution as it's populated by default in the query. This is not intentional and needs to be fixed for the reason that you stated.

**Comment by mnoorenberghe** [comment]
Great, thanks!

**Comment by mnoorenberghe** [comment]
@lmandel, do you use the master branch for staging changes? If not, perhaps the default branch should be gh-pages and master should be removed? It seems like this changed wasn't deployed to gh-pages yet.

**Comment by lmandel** [comment]
I am using master for staging. Just working to cleanup the bugs that shouldn't be on the list (dups, wontfix) before pushing so as not to cause any panic with a big spike in numbers on the boards.

**Comment by mnoorenberghe** [comment]
sounds good
